traffic_portal/app/src/common/modules/table/serverCapabilityServers/TableServerCapabilityServersController.js

```diff
@@ -66,6 +66,35 @@ var TableServerCapabilityServersController = function(serverCapability, servers,
 		locationUtils.navigateToPath('/servers/' + id);
 	};
 
+	$scope.selectServers = function () {
+		var modalInstance = $uibModal.open({
```

Comment:
this should be using `let` or `const` instead of hoisting the declaration with `var` - and really it should be `const` because it's never reassigned.